db/migrate/20180418152139_delete_phone_number.rb

```diff
@@ -0,0 +1,5 @@
+class DeletePhoneNumber < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :users, :phone_number
```

Comment:
Provide the column type to be able to revert this migration
http://guides.rubyonrails.org/active_record_migrations.html#using-the-change-method